xbmc/cores/VideoRenderers/OverlayRenderer.cpp

```diff
@@ -328,12 +328,27 @@ COverlay* CRenderer::Convert(CDVDOverlaySSA* o, double pts)
 {
   CRect src, dst;
   g_renderManager.GetVideoRect(src, dst);
-
-  int width  = MathUtils::round_int(dst.Width());
-  int height = MathUtils::round_int(dst.Height());
-
+  RESOLUTION_INFO res = g_graphicsContext.GetResInfo();
+
+  int frameWidth = res.iWidth;
```

Comment:
1. have discussed with @jmarshallnz and @elupus , and they approved to use GetResInfo() and ALIGN_SCREEN.
2. This only affect the base line subs in ssa. not affect subs which defined position, they still fit by video.